conf/ocsci/independent_rhcs_cluster.yaml

```diff
@@ -0,0 +1,12 @@
+DEPLOYMENT:
+  independent_mode: True
+  force_download_client: True
+
+ENV_DATA:
+  storage_cluster_name: 'ocs-external-storagecluster'
```

Comment:
Until https://bugzilla.redhat.com/show_bug.cgi?id=1852287 is fixed, we'll need to use 'ocs-independent-storagecluster' as the cluster name for the independent mode dashboard to be enabled in the UI

Reply:
ack, one of the dev suggested to hace ```ocs-external-storagecluster``` but that didn't work as expected, changing it to ```ocs-independent-storagecluster``` worked and UI shows it as ```independent```. I ll push this change.

Reply:
done